src/nagini_translation/sif/resources/all.sil

```diff
@@ -4,11 +4,23 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
-import "bool.sil"
+domain SIFDomain[T] {
+    function Low(x: T): Bool
+}
+
+import "../../resources/bool.sil"
+import "../../resources/bytes.sil"
+import "../../resources/iterator.sil"
+import "list.sil"
```

Comment:
Figure out what to do with the others.
Oh, can we redefine Low() to be True in the normal version and keep everything else the same?